internal/handler/common/find.go

```
@@ -79,5 +79,7 @@ func GetFindParams(doc *types.Document, l *zap.Logger) (*FindParams, error) {
 		return nil, err
 	}
 
+	// check AwaitData=true, Tailable=false?
```

It should return the error